distributor-api-v1/changelog.md

```diff
@@ -2,7 +2,5 @@
 
 ## 26.02.2020
 
-* Added documentation for `hotels/getPaymentConfiguration`
-* Deleted old/legacy `payments/getPaymentGateway`
-* [PR](https://github.com/MewsSystems/gitbook-distributor-guide/pull/24)
-
+* Added documentation for `hotels/getPaymentConfiguration`.
+* Deleted old/legacy `payments/getPaymentGateway` [PR](https://github.com/MewsSystems/gitbook-distributor-guide/pull/24).
```

Comment:
@dobras Why was this changed? There should at least be a dash or something, imagine if the **PR** text wouldn't be a link, then the sentence doesn't make sense.

Reply:
1. There were missing `.`
2. The PR should belong to both right? When it was like that it looked like it does not belong to anything. Lonely. Maybe we should put it to both?

Reply:
Other changes seem otherwise ok, so I've merged it. We can decide on the format of the changelog after merging this as 📝